multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
I'm still very skeptical of the absolute import plan. This code could easily need to be moved later, as we would like to gradually disassemble the monolithic "google" namespace API library into components. "requests", "urllib3" and other open-source projects all use relative imports, and as a result they can be embedded into other packages or vendored without modification. As far as nailing down the package name, requests, urllib3 and other libraries show that nailing down the package name is not desirable (requests itself vendors urllib3 for versioning reasons and relative imports makes this possible).

More generally I feel, philosophically, that package structure is a separate concern from the code and the code should operate independently of that as much as possible.  I also feel it's easier to read (explicit) relative imports because the current name of this package is long and only slightly differs from the name of other related packages. Local imports would visually pop out compared to external imports.

If you feel it's critical for the readability review then I will implement the change, but this is the only one of the changes we've discussed that I definitely do not feel good about.